public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -30,13 +30,62 @@ function Keyboard ($) {
         $(document).bind('keyup', documentKeyUp);
         $(document).bind('keydown', documentKeyDown);
     }
+    function toRadians (angle) {
+        return angle * (Math.PI / 180);
+    }
+    // Move in the direction of a link closest to a given angle
```

Comment:
Small thing, but I prefer a comment before a method name to be blocked by /** */ . JSDoc automatically parses this and generates documentation this way.